README.md

```diff
@@ -147,160 +154,237 @@ print("Training set accuracy = " + str(evaluator.evaluate(predictionAndLabels)))
+# move to a permanent place for future use
+dbfs_model_path = 'dbfs:/models/model-full.h5'
+dbutils.fs.cp('file:/tmp/model-full.h5', dbfs_model_path)  
+```
 
-    ```python
-    from keras.applications import InceptionV3
+Now on the prediction side:
 
-    model = InceptionV3(weights="imagenet")
-    model.save('/tmp/model-full.h5')
-    ```
 
-    Now on the prediction side, we can do:
+```python
+from keras.applications.inception_v3 import preprocess_input
+from keras.preprocessing.image import img_to_array, load_img
+import numpy as np
+from pyspark.sql.types import StringType
+from sparkdl import KerasImageFileTransformer
+
+def loadAndPreprocessKerasInceptionV3(uri):
+  # this is a typical way to load and prep images in keras
+  image = img_to_array(load_img(uri, target_size=(299, 299)))  # image dimensions for InceptionV3
+  image = np.expand_dims(image, axis=0)
+  return preprocess_input(image)
+
+dbutils.fs.cp(dbfs_model_path, 'file:/tmp/model-full-tmp.h5')
+transformer = KerasImageFileTransformer(inputCol="uri", outputCol="predictions",
+                                        modelFile='/tmp/model-full-tmp.h5',  # local file path for model
+                                        imageLoader=loadAndPreprocessKerasInceptionV3,
+                                        outputMode="vector")
+
+files = ["/dbfs" + str(f.path)[5:] for f in dbutils.fs.ls(sample_img_dir)]  # make "local" file paths for images
+uri_df = sqlContext.createDataFrame(files, StringType()).toDF("uri")
+
+keras_pred_df = transformer.transform(uri_df)
+```
 
-    ```python
-    from keras.applications.inception_v3 import preprocess_input
-    from keras.preprocessing.image import img_to_array, load_img
-    import numpy as np
-    import os
-    from sparkdl import KerasImageFileTransformer
 
-    def loadAndPreprocessKerasInceptionV3(uri):
-        # this is a typical way to load and prep images in keras
-        image = img_to_array(load_img(uri, target_size=(299, 299)))
-        image = np.expand_dims(image, axis=0)
-        return preprocess_input(image)
+```python
+display(keras_pred_df.select("uri", "predictions"))
+```
 
-    transformer = KerasImageFileTransformer(inputCol="uri", outputCol="predictions",
-                                            modelFile="/tmp/model-full.h5",
-                                            imageLoader=loadAndPreprocessKerasInceptionV3,
-                                            outputMode="vector")
+#### Applying deep learning models at scale to tensors
+Deep Learning Pipelines also provides ways to apply models with tensor inputs (up to 2 dimensions), written in popular deep learning libraries:
+* TensorFlow graphs
+* Keras models
 
-    files = [os.path.abspath(os.path.join(dirpath, f)) for f in os.listdir("/data/myimages") if f.endswith('.jpg')]
-    uri_df = sqlContext.createDataFrame(files, StringType()).toDF("uri")
+##### For TensorFlow users
 
-    final_df = transformer.transform(uri_df)
-    ```
+```python
```

Comment:
better to put imports in the cell they are used in, since people are likely to try out copying a cell at a time